oracles/oracle_transactions.md

```diff
@@ -17,7 +17,7 @@ The transaction contains:
 - Response format definition
 - Query fee (that should be paid for posting a query to the oracle).
 - A TTL (relative in number of blocks, or absolute block height)
-- Transaction fee (must be larger than a minimum proportional to the TTL)
+- Transaction fee. It must be greater than or equal to the mimimum transaction fee plus a component proportional to the relative TTL (quotient of the Euclidean division between the number of blocks and 1000, plus 1 if non-zero remainder).
```

Comment:
The description is straight forward but including an example would be nice anyway

Reply:
@knarz Addressed in new commit.